### PR TITLE
Add info on using custom network plugins

### PR DESCRIPTION
**Pull request by catherineluse**
This PR corresponds to this issue https://github.com/rancher/docs/issues/1539

### CODE REVIEW

**Comment by jiaqiluo** [comment]
LGTM

**Comment by cjellick** [comment]
@superseb maybe if you review for accuracy, @alena1108's review can be quicker

File: content/rke/latest/en/config-options/add-ons/network-plugins/_index.md [hunk 1]
```diff
@@ -92,3 +92,215 @@ network:
 #### Weave encryption
 
 Weave encryption can be enabled by passing a string password to the network provider config.
+
+
+## Installing a Custom Network Plug-in
+
+_Available as of v2.2.4_
```
Comment (superseb) [inline content/rke/latest/en/config-options/add-ons/network-plugins/_index.md]: These are RKE docs and this is a Rancher version. We should make RKE specific docs and then have a Rancher docs section if we need Rancher specific configuration. We have a section where we describe how to use RKE config file options in Rancher here https://rancher.com/docs/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/#config-file

File: content/rke/latest/en/config-options/add-ons/network-plugins/_index.md [hunk 2]
```diff
@@ -92,3 +92,215 @@ network:
...
+    flannel_backend_type: "vxlan"
+  plugin: "canal"
+```
+to:
+```
+network:
+    plugin: none
+```
+
+Then, in the `addons` section of the YAML, add the entire `configmap` file of a cluster that has the network plugin-that you want.
```
Comment (superseb) [inline content/rke/latest/en/config-options/add-ons/network-plugins/_index.md]: It is uploaded as a configmap, but the user doesnt have to supply a configmap. We can re-use definitions/text from https://rancher.com/docs/rke/latest/en/config-options/add-ons/user-defined-add-ons/, as we basically use this functionality to add the necessary network resources.

**Comment by superseb** [comment]
Reviewed

File: content/rke/latest/en/config-options/add-ons/network-plugins/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rke/latest/en/config-options/add-ons/network-plugins/_index.md]: Moved the content to the Rancher docs

File: content/rke/latest/en/config-options/add-ons/network-plugins/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rke/latest/en/config-options/add-ons/network-plugins/_index.md]: Used some copy from the user-defined add-ons page to explain the custom network plug-in

**Comment by catherineluse** [comment]
I had trouble resolving the merge conflicts on this PR. So I responded to the feedback on this other PR with a different branch. https://github.com/rancher/docs/pull/1675

**Pull request closed without merging**